### PR TITLE
Fix return key not working in multiline field

### PR DESCRIPTION
**Pull request by daviserez**

### CODE REVIEW

File: client/app/card/card.component.ts [hunk 1]
```diff
@@ -199,15 +199,6 @@ export class CardComponent extends NaturalAbstractController implements OnInit,
         ...quillConfig.modules,
         keyboard: {
             bindings: {
-                enter: {
```
Comment (lucaguindani) [inline client/app/card/card.component.ts:202]: @PowerKiKi was this change intentional ?

Comment (PowerKiKi) [reply client/app/card/card.component.ts:202]: No, that was most likely a copy-paste mistake from d17d3dfb9067e4286a8a7ec8c74912a2d29b06af. IMHO this PR should be merged.
